app/services/assessments/get_latest_assessment_logs.rb

```diff
@@ -0,0 +1,12 @@
+module Assessments
+  class GetLatestAssessmentLogs
+    class << self
+      def call(user:, installation_id:)
+        AssessmentLog.select('DISTINCT ON (api_name) api_name, *')
+          .where(account_id: user.account_id, installation_id:)
+          .or(AssessmentLog.where(incognia_signup_id: user.incognia_signup_id))
+          .order(api_name: :asc, created_at: :desc)
+      end
```

Comment:
This query occurs on every AssessForm submission right?
Do we need indexes? Or the indexes would be created as a response if lag occurs?

Reply:
Yes! We studied the possibility of an index, but as long as there is an `OR` in this query Postgres can't use any of them. :grimacing: 

Considering that is just for demo purposes, we let it with no indexes.

Reply:
Got it 👍🏼